src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
Ok, I'll take a look at passing the tests without the shim functions tomorrow.
But the 'group' field should be changed nevertheless ;)